pkg/storage/etcd/etcd_helper.go

```diff
@@ -283,7 +304,11 @@ func (h *etcdHelper) extractObj(response *etcd.Response, inErr error, objPtr run
 		}
 		return "", nil, fmt.Errorf("unable to locate a value on the response: %#v", response)
 	}
-	body = node.Value
+
+	body, err = h.transformer.ReadString(node.Value)
+	if err != nil {
+		return body, nil, err
```

Comment:
Yeah, was going to be another commit in this PR once I got the basic "ew, gross" reactions out of the way.